src/config.ts

```diff
@@ -1222,4 +1231,23 @@ export default {
    * there can be in that history.
    */
   BUFFERED_HISTORY_MAXIMUM_ENTRIES: 200,
+
+  /**
+   * Minimum buffer (in seconds ) we should have, regardless of memory
+   * constraints
+   */
+  MIN_BUFFER_LENGTH : 5,
+
+  /**
+   * Minimum distance (in seconds) from the end
+   * we should be able to download
+   * Before starting to garbage collect previous segments
```

Comment:
I'm not sure I understand those sentences

Reply:
Changed